### ISSUE TITLE
Error creating Python code from Pyth Source

### ISSUE THREAD

**Issue by Sobin235** [isaacg1/pyth#242]
I was working on [this problem](https://codegolf.stackexchange.com/questions/142655/draw-my-downslashes), and I created a very basic starting point.

As I tried executing the code, there were more errors than I'd expected, including "parse_token", which I didn't have in my code. I opened the debugger to see what I did, and it did not show the converted python code, just lines on lines of compiler errors.

The code was `VzIqHN=Z?qJ\\hZtZ)+*dZN=HN`, [link is here](https://pyth.herokuapp.com/?code=VzIqHN%3DZ%3FqJ%5C%5ChZtZ%29%2B%2adZN%3DHN&input=%5C%5C%5C%2F%5C%2F%5C%5C%5C%2F%2F%2F%2F%2F&debug=1)

**Comment by cykapup** [comment]
Issue was I typed "J" instead of "H" on character 11, I guess the auto-init tripped up the compiler

**Comment by Maltysen** [comment]
Yeah, this sort of error usually means that you messed up the arity somewhere. This could be because you actually gave the wrong number of arguments, or as you did, used a wrong command, which messed stuff up down the line.

We may want to look into catching these errors and doing our own compile-time error reporting, but that may be hard.
